dataclients/kubernetes/crd.md

```diff
@@ -0,0 +1,596 @@
+# RouteGroup Proposal
+
+Date: 2019-09-04
+Status: open
+Owners: @aryszka @szuecs
+
+This proposal will be kept up to date, if you like to discuss it
+please use https://github.com/zalando/skipper/issues/660
+
+Disclaimer:
+Skipper's `kubernetes` dataclient will also support Ingress in the
+future.
+
+## Purpose
+
+The purpose of the document is to define a CRD, that solves issues
+we observed, while using Ingress.
+
+There are some other solutions out there, like Istio, SMI and others,
+but all of these have drawbacks for skipper provided use cases. One
+common example is an application that needs a redirect to somewhere
+else for a specific path in an Ingress type of object. This example is
+only provided with multiple objects or annotations, both are unsafe
+for concurrent use.
+
+## Problem statement
+
+Provide a format for routing input that makes it easier to benefit
+more from Skipper's routing features, without workarounds, than the
+generic Ingress specs. Primarily targeting more complex Kubernetes
+Ingress scenarios.
```

Comment:
This reads more like the beginning of a solution. The problem statement should IMO make it clear how we provide features using multiple ingresses and why this is a problem.

This makes it more clear that a single resource is a benefit in the proposed solution section.

Reply:
Changed the purpose to make it more focused on us